### ISSUE TITLE
Prestashop 1.7.3.2 Intermittent save

### ISSUE THREAD

**Issue by raphgithub** [PrestaShop/PrestaShop#10518]
**Describe the bug**
When modifying/adding additional content to product: prestashop shows it as "saved" and then the change is not reflected in the Front, and when reloading the BO the change does not appear.

**To Reproduce**
1. Go to 'Catalog>Products'
2. Click on 'one Product, then on description then change the description, hit save'

**Screenshots**
If applicable, add screenshots or screenrecords to help explain your problem.

**Additionnal information**
PrestaShop version: 1.7.3.2
PHP version: 7.0.30-0+deb9u1

**Comment by khouloudbelguith** [comment]
Hi @raphgithub,

Could you please check those issues: #9677, #9954, #9717
It can be an issue similar.
Thanks to check & feedback.

Best regards, Khouloud

**Comment by raphgithub** [comment]
I have tried several browsers, still same intermittent save

**Comment by raphgithub** [comment]
Thanks Khouloud, I'll check and revert

**Comment by khouloudbelguith** [comment]
Hi @raphgithub,

Thanks!
Waiting for your feedback.

Best regards, Khouloud

**Comment by raphgithub** [comment]
Hi @khouloudbelguith,

My issue was different from the ones you mentionned. When saving a simple description of a product: it showed the green notification "changes saved" but did not reflect it Front end.
I did disable the Smarty caching and it seems to be solved now.

Kind regards

Raph

**Comment by khouloudbelguith** [comment]
HI @raphgithub,

Could you please provide me the exact setting that you have made in the performance page.

Best regards, Khouloud

**Comment by raphgithub** [comment]
Hi @khouloudbelguith,

I did what rdubill said on the following page On 01/08/2017 at 2:52 PM, : https://www.prestashop.com/forums/topic/604058-i-cant-save-changes-to-my-products-in-prestashop-17/
Kind regards
Raph

Quote"
To anyone who might be following this - late yesterday afternoon (7/31/17) I went into Advanced Parameters>Performance and changed the following.

1) Left SMARTY alone - that is left caching type at file system and checked never recompile template files.

Towards the bottom you will see a heading of CACHING...

I set use cache to no.

I'm not sure how the setting works in conjunction (if at all) SMARTY - or what impact (if any) this will have on my site's response times.

Since I made this change everything I save is updating instantly.

I'll watch this as I move forward with development.

thanks to all for your input..."
